data/agrima.mdx

```diff
@@ -0,0 +1,18 @@
+---
+name: Agrima
+description:The Company is working on Artificial Intelligence for the last couple of years and had developed a deep learning computer vision technology that could be used for building advanced image recognition solutions. We have used this technology in one of our Apps called Recipe Book. 
+founded: July 2012
+based: Both
+domain: IT Services and IT Consulting
+logo: agrima.png
```

Comment:
@dhananjay425 The logo filename is wrong

Reply:
ohh i will correct that

Reply:
fixed